Kubernetes/omsagent.yaml

```diff
@@ -482,4 +523,67 @@ spec:
       configMap:
         name: container-azm-ms-agentconfig
         optional: true
-
+    - name: azurefile-pv
+      persistentVolumeClaim:
+        claimName: azurefile
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: replicaset-service
```

Comment:
Perfer a more 'spcfici' name - so when anyone lists the servcies in the cluster, they can make out that this is for azure monitor containers.